packages/react/src/utils/useScrollLock.ts

```diff
@@ -52,6 +53,7 @@ function preventScrollStandard(referenceElement?: Element | null) {
       overflowY: html.style.overflowY,
       overflowX: html.style.overflowX,
     };
+    originalHtmlScrollBehavior = html.style.scrollBehavior;
```

Comment:
Why isn't it a part of `originalHtmlStyles`?

Reply:
It has to be assigned after the `.scrollTop/Left` assignments